README.md

```diff
@@ -9,13 +9,13 @@
 
 ## Description
 
-Patch-level verification for [bundler].
+The best tool to find leaky gems in your dependencies. Make sure memory leaks
+are not in your gem dependencies.
 
 ## Features
 
-* Checks for memory leaks of gems in `Gemfile.lock`.
-* Prints memory leak information.
-* Does not require a network connection.
```

Comment:
Why this ` Does not require a network connection.` was removed?

Reply:
@bronzdoc I don't think it's accurate, I guess it doesn't require network connection after doing `gem install bundler-leak` ... 

So, I don't know, is this line here to describe that when you run `bundler-leak` it won't connect to a remote server to fetch the ruby mem advisory db?

Reply:
> So, I don't know, is this line here to describe that when you run bundler-leak it won't connect to a remote server to fetch the ruby mem advisory db?

Yes, bundler-leak has it's own local copy